app/src/main/java/com/vultisig/wallet/ui/screens/ChainCoinScreen.kt

```diff
@@ -310,26 +343,32 @@ private fun CoinItem(
             Spacer(modifier = Modifier.width(16.dp))
             Text(
                 text = title,
-                style = Theme.montserrat.headlineSmall,
+                style = Theme.montserrat.titleMedium,
                 color = appColor.neutral100
             )
             Spacer(modifier = Modifier.weight(1f))
             Text(
                 text = balance,
-                style = Theme.menlo.titleLarge,
+                style = Theme.menlo.titleMedium,
                 color = appColor.neutral0
             )
         }
         Spacer(modifier = Modifier.height(8.dp))
         Text(
             text = fiatBalance,
-            style = Theme.menlo.titleSmall,
+            style = Theme.menlo.titleMedium,
             color = appColor.neutral100,
             modifier = Modifier.padding(top = dimens.marginSmall)
         )
     }
 }
 
+@Preview(showBackground = true, name = "chain coin screen")
+@Composable
+fun ChainCoinScreenPreview() {
```

Comment:
We can keep previews private, as they're never used in a project, so they don't pollute our global namespace.

```suggestion
private fun ChainCoinScreenPreview() {
```